ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobPropertiesSerDe.java

```diff
@@ -76,4 +114,23 @@ public static void putBlobPropertiesToBuffer(ByteBuffer outputBuffer, BlobProper
     Utils.serializeNullableString(outputBuffer, properties.getOwnerId());
     Utils.serializeNullableString(outputBuffer, properties.getServiceId());
   }
+
+  /**
+   * Serialied {@link BlobProperties} to buffer in version {@link #Version2}
+   * @param outputBuffer the {@link ByteBuffer} to write the {@link BlobProperties}
+   * @param properties the {@link BlobProperties} to be serialized
+   */
+  public static void putBlobPropertiesToBufferV2(ByteBuffer outputBuffer, BlobProperties properties) {
```

Comment:
another method is just for the interim period. Once we start to write blobproperies in new format, we may never require the other method.